backend/src/openarchiefbeheer/zaken/tasks.py

```diff
@@ -36,13 +34,12 @@ def configure_retry(client: APIClient) -> APIClient:
 
 
 def retrieve_and_cache_zaken(is_full_resync=False):
-    zrc_service = get_service(APITypes.zrc)
-    zrc_client = build_client(zrc_service)
-    zrc_client = configure_retry(zrc_client)
-
-    config = APIConfig.get_solo()
-    service = config.selectielijst_api_service
-    selectielijst_api_client = build_client(service) if service else None
+    # TODO: We should probably let this error out,
+    # But I don't want to change the behaviour for now while fixing #867
```

Comment:
Another TODO: Almost all non-paginated responses have Etag headers. We should put them to use.

Reply:
I will make an issue also for this :smile:

Reply:
https://github.com/maykinmedia/open-archiefbeheer/issues/893